build.zig.zon

```diff
@@ -1,10 +1,13 @@
 .{
     .name = "zshuffle",
     .version = "0.0.0",
+    .paths = .{
+        "",
+    },
     .dependencies = .{
         .zkwargs = .{
```

Comment:
> If you're using zshuffle actively, I should probably put together a commit to remove zkwargs as a dependency; it's overkill for the little bit of comptime specialization we're doing, and I think zshuffle would have slightly easier to understand code and less toil on your end if I made that tweak.

Sounds like a decent change. It's a bit of a faff to get everything going with all of the breaking changes as Zig evolve.

It has been a very good reference on how to manage dependencies/sub-dependencies in a project to base my own `build.zig` around though!

Reply:
The chance to see how the new package management worked was admittedly my motivation for linking the projects. I'll go ahead and approve this and let you know when I start on that update.